source/install/desktop-app-install.rst

```diff
@@ -18,159 +18,189 @@ Desktop application install guides
 
-Ubuntu and Debian-based systems
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+      - 32-bit systems:
+      
+        .. code-block:: none
+        
+          sudo rpm -i mattermost-desktop-5.0.4-linux-i686.rpm
 
-.. tabs::
+    3. Run Mattermost as a desktop app.
 
-  .. tab:: Install as a Deb Package
+    **Manually update the Desktop App**
 
-    Unofficial, community-driven ``.deb`` packages are available.
+    - 64-bit systems:
+    
+      .. code-block:: none
 
-    1. Download the latest version of the Mattermost Desktop App:
+        sudo rpm -u mattermost-desktop-5.0.4-linux-x86_64.rpm
 
-      - 64-bit systems `mattermost-desktop-5.0.4-linux-amd64.deb <https://releases.mattermost.com/desktop/5.0.4/mattermost-desktop-5.0.4-linux-amd64.deb>`__
-      - 32-bit systems `mattermost-desktop-5.0.4-linux-i386.deb <https://releases.mattermost.com/desktop/5.0.4/mattermost-desktop-5.0.4-linux-i386.deb>`__
+    - 32-bit systems:
+    
+      .. code-block:: none
+ 
+        sudo rpm -u mattermost-desktop-5.0.4-linux-i686.rpm
 
-    2. At the command line, execute one of the following commands depending on the package that you downloaded:
+    .. tip:: 
+      You can review the current version of your Desktop App by selecting the **More** |more-icon-vertical| icon located in the top left corner of the Mattermost window, then selecting **Help > Version...**.
 
-      - 64-bit systems ``sudo dpkg -i mattermost-desktop-5.0.4-linux-amd64.deb``
-      - 32-bit systems ``sudo dpkg -i mattermost-desktop-5.0.4-linux-i386.deb``
+  .. tab:: Generic Linux
 
-    3. Run Mattermost as a desktop app.
+    A beta AppImage distribution of a compressed tarball is available. Automatic app updates aren’t supported. You must update your app manually. 
 
+    1. Download the latest version of the Mattermost Desktop App:
 
-  .. tab:: Install from Mattermost Repository
+      - 64-bit systems: `mattermost-desktop-5.0.4-linux-x64.tar.gz <https://releases.mattermost.com/desktop/5.0.4/mattermost-desktop-5.0.4-linux-x64.tar.gz>`__
+      - 32-bit systems: `mattermost-desktop-5.0.4-linux-ia32.tar.gz <https://releases.mattermost.com/desktop/5.0.4/mattermost-desktop-5.0.4-linux-ia32.tar.gz>`__
 
-    Community-driven ``.deb`` packages are published into Mattermost Repository.
-    
-    1. In a terminal window, run the following command to configure the Mattermost Repository.
+    2. Extract the archive to a convenient location, then execute ``mattermost-desktop`` located inside the extracted directory.
 
-      ``curl -o- https://deb.packages.mattermost.com/setup-repo.sh | sudo bash``
+    3. To create a Desktop launcher, open the file ``README.md``, and follow the instructions in the **Desktop launcher** section.
 
-    2. Install the Mattermost Desktop package.
+  .. tab:: macOS
 
-      ``sudo apt install mattermost-desktop``
+    MacOS 10.15+ is required. You have two ways to install the Desktop App, and how you install the app determines whether it updates automatically.
```

Comment:
Small thing, should be "macOS"

Reply:
Given that this is the first word in a sentence, I've elected to apply sentence case.